comfy_cli/jobs_state.py

```
@@ -0,0 +1,181 @@
+    """Read a state file. Returns None if the file doesn't exist."""
+    path = state_path(prompt_id)
+    if not path.exists():
+        return None
+    try:
+        data = json.loads(path.read_text(encoding="utf-8"))
+    except (OSError, json.JSONDecodeError):
+        return None
+    if not isinstance(data, dict):
+        return None
+    # Tolerant load: drop unknown keys, fill defaults for missing ones.
+    known = {f.name for f in JobState.__dataclass_fields__.values()}
```

Pylint (4.0.5)

[error] Class 'JobState' has no '__dataclass_fields__' member

(E1101)